packages/cozy-stack-client/src/OAuthClientsCollection.js

```diff
@@ -104,17 +104,13 @@ class OAuthClientsCollection extends DocumentCollection {
    * Destroys the OAuth client on the server
    *
    * @param {io.cozy.oauth.clients} client     The client document to destroy
-   * @param {string}                client._id The client's id
-   *
    * @returns {{ data }} The deleted client
    */
-  async destroy({ _id }) {
-    const resp = await this.stackClient.fetchJSON(
-      'DELETE',
-      uri`/settings/clients/${_id}`
-    )
+  async destroy(client) {
```

Comment:
nit: `client` is generally referring to a cozy-client instance, I think it would avoid confusion to name the parameter `oauthClient` for instance ;)